src/daemon/daemon.cpp

```diff
@@ -1249,17 +1250,17 @@ try // clang-format on
     mpl::ClientLogger<RecoverReply> logger{mpl::level_from(request->verbosity_level()), *config->logger, server};
 
     const auto instances_and_status =
-            find_requested_instances(request->instance_names().instance_name(), deleted_instances,
-                                     std::bind(&Daemon::check_instance_exists, this, std::placeholders::_1));
+        find_requested_instances(request->instance_names().instance_name(), deleted_instances,
+                                 std::bind(&Daemon::check_instance_exists, this, std::placeholders::_1));
     const auto& instances = instances_and_status.first; // use structured bindings instead in C++17
     const auto& status = instances_and_status.second;   // idem
 
-    if(status.ok())
+    if (status.ok())
```

Comment:
I'm not sure how granular `clang-format` is wrt to changes within a file, but it's always a pain when `clang-format` makes changes unrelated to any of the changes.  I'm not so concerned about `git blame` here since unless the original author makes the change in a future `clang-format`, then it will be messed up then too.  That said, I'm fine with either reverting this or leaving it.

Reply:
I believe git-clang-format only changes lines with a plus in a diff. Plain clang-format would process the whole file though

Reply:
Leaving this alone: I don't want to mess with existing commits and no point recording a new one